roles/kubernetes-apps/ansible/defaults/main.yml

```diff
@@ -1,5 +1,6 @@
 ---
 # Limits for coredns
+dns_cpu_limit: 300m
```

Comment:
It can be changed to 
```
dns_cpu_limit: 0 
```
to aviod change the default value.

Reply:
Ok thank you, I made the change.